backend/postgres/store.go

```diff
@@ -32,6 +31,7 @@ func NewStore(connectURL string) backend.Store {
 
 	//db.Exec(`ALTER TABLE shares ADD CONSTRAINT shares_unique UNIQUE (user_id, basket_id)`)
 	_, _ = db.Exec(`ALTER TABLE offers ADD CONSTRAINT offer__user_task__unique UNIQUE (user_id, task_id)`)
+	_, _ = db.Exec(`ALTER TABLE tasks  ADD CONSTRAINT tasks_date_sequence CHECK(from_date <= to_date)`)
```

Comment:
```suggestion
	_, _ = db.Exec(`ALTER TABLE tasks ADD CONSTRAINT tasks_date_sequence CHECK(from_date <= to_date)`)
```
После tasks лишний пробел убрал просто, чтобы попробовать комментарии оставлять :)

Reply:
Ты вроде можешь одной кнопкой закоммитить это изменение!

Reply:
Иправила, закоммитила